lib/generators/connector/templates/entity.rb

```diff
@@ -34,23 +34,37 @@ def update_external_entity(mapped_connec_entity, external_id, external_entity_na
   # e.g entity['id']
   def self.id_from_external_entity_hash(entity)
     # TODO
+    # This method return the id from external_entity_hash
```

Comment:
I think you duplicated the doc here? See #183

Reply:
ya my bad, rebase conflict was resolved wrong, fixed it